detectors/resources/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'java'
+    id 'com.github.sherter.google-java-format' version '0.9'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation "com.github.tomakehurst:wiremock-jre8:2.27.2"
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
Should versions of the dependencies be extracted to the top gradle file (like constants below)?